R/io-csv-functions.R

```diff
@@ -112,38 +110,39 @@
 #' lazy_frame$collect()
 #' unlink(my_file)
 pl__scan_csv <- function(
-    source,
-    ...,
-    has_header = TRUE,
-    separator = ",",
-    comment_prefix = NULL,
-    quote_char = '"',
-    skip_rows = 0,
-    schema = NULL,
-    schema_overrides = NULL,
-    null_values = NULL,
-    missing_utf8_is_empty_string = FALSE,
-    ignore_errors = FALSE,
-    cache = FALSE,
-    infer_schema = TRUE,
-    infer_schema_length = 100,
-    n_rows = NULL,
-    encoding = c("utf8", "utf8-lossy"),
-    low_memory = FALSE,
-    rechunk = FALSE,
-    skip_rows_after_header = 0,
-    row_index_name = NULL,
-    row_index_offset = 0,
-    try_parse_dates = FALSE,
-    eol_char = "\n",
-    raise_if_empty = TRUE,
-    truncate_ragged_lines = FALSE,
-    decimal_comma = FALSE,
-    glob = TRUE,
-    storage_options = NULL,
-    retries = 2,
-    file_cache_ttl = NULL,
-    include_file_paths = NULL) {
+  source,
+  ...,
+  has_header = TRUE,
+  separator = ",",
+  comment_prefix = NULL,
+  quote_char = '"',
+  skip_rows = 0,
+  schema = NULL,
+  schema_overrides = NULL,
+  null_values = NULL,
+  missing_utf8_is_empty_string = FALSE,
+  ignore_errors = FALSE,
+  cache = FALSE,
+  infer_schema = TRUE,
+  infer_schema_length = 100,
+  n_rows = NULL,
+  encoding = c("utf8", "utf8-lossy"),
+  low_memory = FALSE,
+  rechunk = FALSE,
+  skip_rows_after_header = 0,
+  row_index_name = NULL,
+  row_index_offset = 0,
+  try_parse_dates = FALSE,
+  eol_char = "\n",
+  raise_if_empty = TRUE,
+  truncate_ragged_lines = FALSE,
+  decimal_comma = FALSE,
+  glob = TRUE,
+  storage_options = NULL,
+  retries = 2,
+  file_cache_ttl = NULL,
+  include_file_paths = NULL
+) {
```

Comment:
Why?

Reply:
Looks like `styler` dev version changes indentation, reverted for now